src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_account_filter_scenarios.py

```diff
@@ -69,6 +69,8 @@ def test_ams_account_filter_create_and_show(self, storage_account_for_create):
             self.check('tracks[1].trackSelections[1].value', 'MP4A')
         ])
 
+        self.cmd('az ams account delete -n {amsname} -g {rg}')
```

Comment:
The @ResourceGroupPreparer is designed to tear down the resource group at the end of the test.  This should remove the AMS account (assuming it is bound to the RG) and render these changes unnecessary.